db/migrate/20230822155817_create_users.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
```

Comment:
- [ ] It would be better to run the migrations after creating the migration file, currently the schema is missing because we did not run the migration. `rails db:migrate` could create the schema file for us
![Screenshot from 2023-08-22 18-39-55](https://github.com/MarcoDDM/blog-app/assets/46388113/0191b721-4fa4-4e7d-b288-20a5dee4f95d)